cli/commands/terraform/download_source.go

```diff
@@ -21,7 +21,7 @@ import (
 const MODULE_MANIFEST_NAME = ".terragrunt-module-manifest"
 
 // file to indicate that init should be executed
-const moduleInitRequiredFile = ".terragrunt-init-required"
+const MODULE_INIT_REQUIRED_FILE = ".terragrunt-init-required"
```

Comment:
Unlike other languages, in GoLang we use camelcase to name constants. It would be nice to get rid of ALL CAPS someday, especially like this [ones](https://github.com/gruntwork-io/terragrunt/blob/dd602a33aa4a215258bb2dd96e588b7dcb06d9ba/test/integration_test.go#L60-L213).

[Naming convention](https://go.dev/wiki/CodeReviewComments#mixed-caps) 
[Example](https://github.com/aws/aws-sdk-go/blob/02c1f723b528251a45068001bee8b56d904d7484/service/s3/sse.go#L34-L37)

This comment applies to all places where such changes are made.

Reply:
It's strange that `lint` doesn't report that this is incorrect.

Reply:
Could we configure `lint` to report about Golang naming convention violations?

Reply:
Yup! There's a `style` preset. I'll either add that now, or in the next PR. I'm not sure how many findings will pop up.

Reply:
I fixed the issues in this file. Do you mind if I don't search through the rest of the PR for instances like this, and instead resolve it when we get the style lint added?

Reply:
Of course, do as you wish. Thanks!